katportalclient/client.py

```diff
@@ -1213,6 +1213,78 @@ def sensor_detail(self, sensor_name):
         else:
             raise tornado.gen.Return(results[0])
 
+    @tornado.gen.coroutine
+    def sensor_value(self, sensor_name, include_value_ts=False):
+        """Return the latest value of a sensor.
```

Comment:
Don't need to change. Not sure if "sensor_value" is a bit confusing as it actually returns the latest sensor sample - which include the value. But I think it is fine as the example shows how it is used. Could have been something like "get_latest" ?